CHANGELOG.md

```diff
@@ -8,6 +8,17 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
+## [0.2.1] - 2019-11-17
+
+### Added
+
+- Added new `$in_list` parameter to the `PrefixAllGlobalsSniff::process_variable_assignment()` method, so that the method is compatible with the upstream WPCS method (changed in WPCS 2.2.0).
+- Travis: Ignore PHP deprecation notices for stable PHPCS releases. Deprecation notices thrown by already released PHPCS versions won't get fixed anymore (in that version), so failing the unit tests on those is moot and will skew the reliability of the Travis results.
```

Comment:
Comprehensive entry, but irrelevant for users of the library.

IMO a simple: `Travis: minor build script tweaks` to cover both Travis changes would be sufficient.

Just my two pennies. It's up to you to decide how comprehensive you want the changelog to be, so if you want to leave it as is, that's fine too.

Reply:
Yeah, it makes sense. The details are written in the commit message, and in the travis yml file itself :+1: